optimizely/helpers/validator.py

```diff
@@ -151,3 +152,25 @@ def is_user_profile_valid(user_profile):
       return False
 
   return True
+
+
+def is_non_empty_string(test_var):
+  """ Determine if provided input is a non-empty string or not.
+
+  Args:
+    test_var: Variable which needs to be validated.
+
+  Returns:
+    Boolean depending upon whether input is valid or not.
+  """
+  PY3 = sys.version_info[0] == 3
+
+  if PY3:
```

Comment:
What happens if you don't do this?

Reply:
basestring was removed in Python3. https://docs.python.org/3.0/whatsnew/3.0.html

Reply:
I am wondering if there is a better way to accomplish what you are doing here.